docs/api/web-frame-main.md

```diff
@@ -70,6 +70,16 @@ or `undefined` if there is no WebFrameMain associated with the given IDs.
 
 Process: [Main](../glossary.md#main-process)
 
+### Instance Events
+
+#### Event: 'dom-ready'
+
+Returns:
+
+* `event` Event
+
+Emitted when the document is loaded.
```

Comment:
If we're going to add this event, we should probably add all the concomitant navigation events (will-navigate, did-navigate, did-finish-load, did-fail-load, etc etc.)

Reply:
I'm all for this, but would like to consider splitting out that work into separate PRs if possible.